lib/msal-common/src/unifiedCache/generator/TokenCacheGenerator.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AccessTokenEntity } from "../entities/AccessTokenEntity";
+import { IdTokenEntity } from "../entities/IdTokenEntity";
+import { RefreshTokenEntity } from "../entities/RefreshTokenEntity";
+import { AuthenticationResult } from "../../response/AuthenticationResult";
+
+export class TokenCacheGenerator {
+    /**
+     * Create IdTokenEntity
+     * @param homeAccountId
+     * @param authenticationResult
+     * @param clientId
+     * @param authority
+     */
+    static createIdTokenEntity(
```

Comment:
Maybe these static methods should be part of the entity classes themselves?

Reply:
Yes. This is a good idea. Moving them.